net-im/libcommuni/files/uchardet.pri

```diff
@@ -0,0 +1,17 @@
+######################################################################
+# Communi
+######################################################################
+
+include(../pkg.pri)
+
+pkgExists(uchardet) {
+	CONFIG += link_pkgconfig
+	PKGCONFIG += uchardet
+}
+
+isEmpty(PKGCONFIG) {
+	error("UChardet support has been enabled, but the UChardet installation has not been found. Did you emerged app-i18n/uchardet?")
+} else {
+	LIBS += -luchardet
```

Comment:
Aren't you using pkgconfig already? Or am I misguessing the meaning of the above.

Reply:
To be honest, I don't know. I just took `/src/3rdparty/icu/icu.pri` as a template and done it the same way.